src/components/homePage.tsx

```diff
@@ -0,0 +1,9 @@
+export const HomePage = () => {
```

Comment:
The file name should match the name of its primary named or default export. Rename the file to 'HomePage.tsx' instead of 'homePage.tsx'.

Reply:
Not fixed 
![image](https://github.com/mate-academy/react_people-table-basics/assets/99403198/fcdaa496-5d08-4e5e-b0a7-1c7c1b4396c6)

Reply:
Still not fixed. Do you have any blockers here?